crates/sqlexec/src/session.rs

```diff
@@ -57,6 +57,8 @@ pub enum ExecutionResult {
     Commit,
     /// Transaction rolled abck.
     Rollback,
+    /// Data successfully deleted.
+    DeleteSuccess,
```

Comment:
(future): We'll want to also return how many rows were deleted so that we can show that to the user.